### DISCUSSION TITLE
Unable to select font size and localisation language in account settings

### DISCUSSION THREAD

**Discussion by kukoboris** [danny-avila/LibreChat#3769] [Troubleshooting]
When I try to select font size and localisation language in the account settings, scrolling in the options selection menu doesn't work (neither mouse nor keyboard)
<img width="688" alt="изображение" src="https://github.com/user-attachments/assets/1d143712-741e-436e-ac18-7a32ad9d7cfc">
<img width="763" alt="изображение" src="https://github.com/user-attachments/assets/d70c567b-57e4-42c4-84f1-a7b364492296">

**Comment by danny-avila** [comment]
Thanks, I think they are just missing accessibility styling, because i can scroll and press Enter to switch but there's no styling to show it. I'm going to move this to issues as a11y

**Reply by danny-avila** [reply]
https://github.com/danny-avila/LibreChat/issues/3770
